packages/quantic/force-app/main/default/lwc/quanticRecentQueries/quanticRecentQueries.css

```
@@ -0,0 +1,28 @@
+.container {
+  width: 100%;
+  margin: 0 2em 0 0;
+}
+
+.query-text {
+  white-space: nowrap;
+  overflow: hidden;
+  text-overflow: ellipsis;
+  color: #0070d2;
+}
+
+li:hover {
+  text-decoration: underline;
+  cursor: pointer;
+}
+
+.icon-color {
+  color: #0070d2;
```

Do we have access to a css variable for this color ?

---

```suggestion
  color: var(--brand-text-link, #0176d3);
```
This should do the trick and be consistent with the recentresultlist color